projects/control-service/projects/helm_charts/pipelines-control-service/templates/deployment.yaml

```diff
@@ -252,6 +252,10 @@ spec:
             - name: DATAJOBS_BUILDER_REGISTRY_SECRET
               value: {{ template "pipelines-control-service.fullname" . }}-dtjb-buldr-rgstry-scrt
             {{- end }}
+            {{- if .Values.controlK8sK8sSupportsV1CronJob }}
+            - name: DATAJOBS_SUPPORT_V1CRONJOB
+            value: "{{ .Values.controlK8sK8sSupportsV1CronJob }}"
```

Comment:
If you are adding new values.yaml field `controlK8sK8sSupportsV1CronJob` you need to document it in values.yaml 

The double k8s seem redundant `controlK8sSupportsV1CronJob` is enough.